docs/recommendations/DNA/alleles.md

```diff
@@ -30,14 +30,17 @@ Allele: a series of variants on one chromosome.
 <tr><th class="section" colspan=2>Variants with unknown or uncertain phase</th></tr>
 <tr>
 <th>Syntax</th>
-<td><code>accession ":" type "." "[" posedit "(;)" posedit"]"</code></td>
+<td><code>sequence_identifier ":" coordinate_type "." "[" position_edit "(;)" position_edit"]"</code></td>
 </tr>
 <tr>
 <th>Examples</th>
 <td><code>NC_000001.11:g.[123G>A(;)345del]</code></td>
 </tr>
 </table>
 
+- `position_edit` = any g. position-edit, such as [deletion](./deletion.md), [insertion](./insertion.md), or [deletion-insertion](./delins.md)
```

Comment:
It's not necessarily `g.` based; this page is also used for the `c.` coordinate type.